engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/JobPreparationActor.scala

```diff
@@ -25,7 +25,9 @@ object JobPreparationActor {
             jobKey: BackendJobDescriptorKey,
             factory: BackendLifecycleActorFactory,
             initializationData: Option[BackendInitializationData]) = {
-    Props(new JobPreparationActor(executionData, jobKey, factory, initializationData)).withDispatcher("akka.dispatchers.slow-actor-dispatcher")
+    // Note that JobPreparationActor doesn't run on the engine dispatcher as it mostly executes backend-side code
+    // (WDL expression evaluation using Backend's expressionLanguageFunctions)
+    Props(new JobPreparationActor(executionData, jobKey, factory, initializationData))
```

Comment:
The default one (same as backends)
This is an artifact of the way wdl expressions are evaluated, which is not ideal but is another discussion (which I'd actually like to make my developer's choice issue). 
At least it works well so it's not a big issue IMO, it's just not as clean as it could be.